packages/connector-aliyun-dm/src/index.ts

```diff
@@ -45,21 +41,38 @@ export default class AliyunDmConnector implements EmailConnector {
       )
     );
 
-    return singleSendMail(
-      {
-        AccessKeyId: accessKeyId,
-        AccountName: accountName,
-        ReplyToAddress: 'false',
-        AddressType: '1',
-        ToAddress: address,
-        FromAlias: fromAlias,
-        Subject: template.subject,
-        HtmlBody:
-          typeof data.code === 'string'
-            ? template.content.replace(/{{code}}/g, data.code)
-            : template.content,
-      },
-      accessKeySecret
-    );
+    try {
+      return await singleSendMail(
+        {
+          AccessKeyId: accessKeyId,
+          AccountName: accountName,
+          ReplyToAddress: 'false',
+          AddressType: '1',
+          ToAddress: address,
+          FromAlias: fromAlias,
+          Subject: template.subject,
+          HtmlBody:
+            typeof data.code === 'string'
+              ? template.content.replace(/{{code}}/g, data.code)
+              : template.content,
+        },
+        accessKeySecret
+      );
+    } catch (error: unknown) {
+      if (error instanceof HTTPError) {
+        const {
+          response: { body: rawBody },
+        } = error;
+        assert(
+          typeof rawBody === 'string',
+          new ConnectorError(ConnectorErrorCodes.InvalidResponse)
+        );
+        const body = singleSendMailErrorResponseGuard.parse(JSON.parse(rawBody));
```

Comment:
the `parse` will throw, and then we lost the response data. I suggest just saving the raw body, because this is used for human reading, parse it or not won't make difference.